src/structured-data/structured-data-tags.js

```diff
@@ -84,7 +84,7 @@ function generateArticleData (structuredData = {}, story = {}, publisherConfig =
     "author": authorData(authors),
     "keywords": metaKeywords,
     "articleBody": (storyKeysPresence && getCompleteText(story)) || '',
-    "dateCreated": stripMillisecondsFromTime(new Date(story['created-at'])),
+    "dateCreated": stripMillisecondsFromTime(new Date(story['first-published-at'])),
     "dateModified": stripMillisecondsFromTime(new Date(story['updated-at'])),
```

Comment:
this would be `last-published-at` right?

Reply:
jeevan and anand said be to use updated at